docs/configuration/module-config/canned-message.mdx

```diff
@@ -10,7 +10,7 @@ import TabItem from "@theme/TabItem";
 
 The Canned Message Module will allow you to send messages to the mesh network from the device without using the phone app. You can predefine text messages to choose from.
 
-The canned message module config options are: Enabled, Save, and Sender. Range Test Module config uses an admin message sending a `ConfigModule.CannedMessage` protobuf.
+The canned message module config options are: Enabled, Send Bell, Messages, Input Source, Rotary Encoder Enabled, Up Down Encoder Enabled, Input Broker Pin A, Input Broker Pin B, Input Broker Pin Press, Input Broker Event Clockwise, Input Broker Event Counter Clockwise, and Input Broker Event Press. Canned Message config uses an admin message sending a `ConfigModule.CannedMessage` protobuf.
```

Comment:
Similar to External Notification, I think this would be easier to read as a list.

Reply:
@charminULTRA Same as EXT Notif.  Just trying to keep things consistent.  This one also happens to currently have incorrect information.  I think it was a cut n paste from another module and the writer forgot to change it.